prog/learn_storage.rb

```diff
@@ -26,26 +26,23 @@ def df_command(path = "") = "df -B1 --output=target,size,avail #{path}"
 
   def make_model_instances
     devices = DfRecord.parse_all(sshable.cmd(df_command))
-    if devices.none? { _1.optional_name }
-      rec = DfRecord.parse_all(sshable.cmd(df_command("/var/storage"))).first
-      [
-        StorageDevice.new_with_id(
-          vm_host_id: vm_host.id, name: "DEFAULT",
-          # reserve 5G the host.
-          available_storage_gib: [rec.avail_gib - 5, 0].max,
-          total_storage_gib: rec.size_gib
-        )
-      ]
-    else
-      devices.filter_map do |rec|
-        next unless (name = rec.optional_name)
-        StorageDevice.new_with_id(
-          vm_host_id: vm_host.id, name: name,
-          available_storage_gib: rec.avail_gib,
-          total_storage_gib: rec.size_gib
-        )
-      end
+    rec = DfRecord.parse_all(sshable.cmd(df_command("/var/storage"))).first
+    sds = [StorageDevice.new_with_id(
+      vm_host_id: vm_host.id, name: "DEFAULT",
+      # reserve 5G the host.
+      available_storage_gib: [rec.avail_gib - 5, 0].max,
+      total_storage_gib: rec.size_gib
+    )]
+
+    devices.filter_map do |rec|
```

Comment:
Since you are not using returned value from this line, you can use `each` instead of `map`.
```suggestion
    devices.each do |rec|
```